src/main/java/com/minecolonies/network/messages/ColonyViewWorkOrderMessage.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.colony.CitizenData;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.CitizenData'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)